Formula/libbinio.rb

```diff
@@ -1,9 +1,9 @@
 class Libbinio < Formula
-  desc "Binary I/O stream class library"
-  homepage "https://libbinio.sourceforge.io/"
-  url "https://downloads.sourceforge.net/project/libbinio/libbinio/1.4/libbinio-1.4.tar.bz2"
-  sha256 "4a32d3154517510a3fe4f2dc95e378dcc818a4a921fc0cb992bdc0d416a77e75"
-  license "LGPL-2.1"
+  desc "Binary I/O stream class library (used by AdPlug)"
+  homepage "https://adplug.github.io/libbinio/"
+  url "https://github.com/adplug/libbinio/releases/download/libbinio-1.5/libbinio-1.5.tar.bz2"
+  sha256 "398b2468e7838d2274d1f62dbc112e7e043433812f7ae63ef29f5cb31dc6defd"
+  license "LGPL-2.1-only"
```

Comment:
```suggestion
  license "LGPL-2.1-or-later"
```
From the code headers:
```
 * This library is free software; you can redistribute it and/or
 * modify it under the terms of the GNU Lesser General Public
 * License as published by the Free Software Foundation; either
 * version 2.1 of the License, or (at your option) any later version.
```